src/command_modules/azure-cli-sql/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.0.16
+++++++
+* Added az sql server dns-alias commands.
```

Comment:
You'll need to rebase and add this to the release notes for the SQL PR that was just merged.